db/range_tombstone_fragmenter.cc

```diff
@@ -56,11 +59,13 @@ FragmentedRangeTombstoneList::FragmentedRangeTombstoneList(
   keys.reserve(num_unfragmented_tombstones_);
   values.reserve(num_unfragmented_tombstones_);
   // Reset the counter to zero for the next iteration over keys.
+  num_unfragmented_tombstones_ = 0;
   total_tombstone_payload_bytes_ = 0;
   for (unfragmented_tombstones->SeekToFirst(); unfragmented_tombstones->Valid();
-       unfragmented_tombstones->Next()) {
-    total_tombstone_payload_bytes_ += unfragmented_tombstones->key().size() +
-                                      unfragmented_tombstones->value().size();
+       unfragmented_tombstones->Next(), num_unfragmented_tombstones_++) {
```

Comment:
I think it's okay to use value size as raw payload bytes here, which keeps the code cleaner and less change to make.

Reply:
db_flush_test will fail:

```
Expected equality of these values:
  mem_data_bytes
    Which is: 8205814
  EXPECTED_MEMTABLE_PAYLOAD_BYTES_AT_FLUSH
    Which is: 194050
```

Reply:
Is the difference here supposed to be how much space sequence number takes? That number seems quite large, a lot more than key + value size?

Reply:
Yes, it is the sequence number space.